internals/types/index.d.ts

```diff
@@ -383,7 +383,7 @@ export interface ProjectLink {
 export interface PaginationOptions {
   prev: number;
   count: number;
-  next?: number;
+  next?: number | null;
```

Comment:
<!-- Questions comments seek clarification.  -->
**question (non-blocking):** ‏Is there some type shuffling we can do to make this not both `null` and `undefined`?

Reply:
From what I've seen so far I think the API will always return `null`. Will double check on that though.

Reply:
We could wrap it to get rid of one of the types. It's not a huge deal, but I do like simplifying these when they come up. I don't want to read the type later and wonder what the difference between `undefined` and `null` means for this property.

Reply:
Updated to use the same type as defined in API repo